modules/navigation.js

```diff
@@ -0,0 +1,29 @@
+export function showBooksSection() {
+  const booksSection = document.getElementById('books-section');
+  const addBookSection = document.getElementById('add-book-section');
+  const contactSection = document.getElementById('contact-section');
+
+  booksSection.style.display = 'block';
+  addBookSection.style.display = 'none';
+  contactSection.style.display = 'none';
+}
+
+export function showAddBookSection() {
+  const booksSection = document.getElementById('books-section');
+  const addBookSection = document.getElementById('add-book-section');
+  const contactSection = document.getElementById('contact-section');
+
+  booksSection.style.display = 'none';
+  addBookSection.style.display = 'block';
+  contactSection.style.display = 'none';
+}
+
+export function showContactSection() {
+  const booksSection = document.getElementById('books-section');
+  const addBookSection = document.getElementById('add-book-section');
+  const contactSection = document.getElementById('contact-section');
+
+  booksSection.style.display = 'none';
+  addBookSection.style.display = 'none';
+  contactSection.style.display = 'block';
+}
```

Comment:
- [ ] You did a great job to implement modules, but you should also consider exporting function in ES6 way than using function key word .  You can check [this ](https://stackoverflow.com/questions/36458697/is-it-possible-to-export-arrow-functions-in-es6-7)on how you can export a function in ES6 and improve your solution

Reply:
Done!!!
![image](https://github.com/lerfast/Awesome-book-ES6-2/assets/111823606/564e3267-8649-4f66-a8a3-93ee090aee31)